<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-22T20:34:40 IST" name="Failed suite [Suite]" finished-at="2023-08-22T20:35:25 IST" duration-ms="45127">
    <groups>
    </groups>
    <test started-at="2023-08-22T20:34:40 IST" name="Test(failed)" finished-at="2023-08-22T20:35:25 IST" duration-ms="45127">
      <class name="com.nopCommerceapplication.testcases.checkingorderstest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.nopCommerceapplication.testcases.checkingorderstest@942a29c]" started-at="2023-08-22T20:35:02 IST" name="setup" finished-at="2023-08-22T20:35:09 IST" duration-ms="6312" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="checkingorders()[pri:0, instance:com.nopCommerceapplication.testcases.checkingorderstest@942a29c]" started-at="2023-08-22T20:35:09 IST" name="checkingorders" finished-at="2023-08-22T20:35:24 IST" duration-ms="15695" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkingorders -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.nopCommerceapplication.testcases.checkingorderstest@942a29c]" started-at="2023-08-22T20:35:24 IST" name="teardown" finished-at="2023-08-22T20:35:25 IST" duration-ms="158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.nopCommerceapplication.testcases.checkingorderstest -->
      <class name="com.nopCommerceapplication.testcases.DDLogintest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.nopCommerceapplication.testcases.DDLogintest@3e11f9e9]" started-at="2023-08-22T20:34:41 IST" name="setup" finished-at="2023-08-22T20:34:47 IST" duration-ms="6156" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="DdLogintest(java.lang.String,java.lang.String)[pri:0, instance:com.nopCommerceapplication.testcases.DDLogintest@3e11f9e9]" started-at="2023-08-22T20:34:47 IST" name="DdLogintest" data-provider="abc" finished-at="2023-08-22T20:34:52 IST" duration-ms="4754" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@yourstore.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://admin-demo.nopcommerce.com/admin/] but found [https://admin-demo.nopcommerce.com/login?returnurl=%2Fadmin%2F]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://admin-demo.nopcommerce.com/admin/] but found [https://admin-demo.nopcommerce.com/login?returnurl=%2Fadmin%2F]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.nopCommerceapplication.testcases.DDLogintest.DdLogintest(DDLogintest.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DdLogintest -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.nopCommerceapplication.testcases.DDLogintest@3e11f9e9]" started-at="2023-08-22T20:34:52 IST" name="teardown" finished-at="2023-08-22T20:34:52 IST" duration-ms="177" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.nopCommerceapplication.testcases.DDLogintest@3e11f9e9]" started-at="2023-08-22T20:34:52 IST" name="setup" finished-at="2023-08-22T20:34:58 IST" duration-ms="6461" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="DdLogintest(java.lang.String,java.lang.String)[pri:0, instance:com.nopCommerceapplication.testcases.DDLogintest@3e11f9e9]" started-at="2023-08-22T20:34:58 IST" name="DdLogintest" data-provider="abc" finished-at="2023-08-22T20:35:02 IST" duration-ms="3485" status="FAIL">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
            <param index="1">
              <value is-null="true"/>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://admin-demo.nopcommerce.com/admin/] but found [https://admin-demo.nopcommerce.com/login?ReturnUrl=%2Fadmin%2F]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://admin-demo.nopcommerce.com/admin/] but found [https://admin-demo.nopcommerce.com/login?ReturnUrl=%2Fadmin%2F]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.nopCommerceapplication.testcases.DDLogintest.DdLogintest(DDLogintest.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DdLogintest -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.nopCommerceapplication.testcases.DDLogintest@3e11f9e9]" started-at="2023-08-22T20:35:02 IST" name="teardown" finished-at="2023-08-22T20:35:02 IST" duration-ms="266" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.nopCommerceapplication.testcases.DDLogintest -->
    </test> <!-- Test(failed) -->
  </suite> <!-- Failed suite [Suite] -->
</testng-results>
